### PR TITLE
Added docs for submitting and working tickets.

### LINKED ISSUE #4: Add docs for making tickets
Expand `docs/TICKETS.md `. 

* Include info on the `.github/ISSUE_TEMPLATE` that are included in the project. 
* Talk about labels
* Talk about assigning yourself to the ticket when you start work on it.

### PR DESCRIPTION
**Pull request by Morpheus636**
Added docs for making tickets and "working" them. 
Feel free to make any changes you think necessary. 

[Issue (#4)](https://github.com/Python-Practice-Discord/Project-Template/issues/4)

### CODE REVIEW

File: docs/TICKETS.md [hunk 1]
```diff
@@ -1,8 +1,85 @@
...
+To submit a bug report or feature request, go to the "Issues" tab in the repository,
+and click "New Issue". Then, select the Bug Report template or the feature request template.
+
+Make sure you include as much of the requested information as you have, since it makes it much
+easier for contributors to investigate.
 
 # Working a Ticket
+Members of the Core team, and the Team Lead should "work" each ticket submitted, by applying the
+proper labels, gathering information, answering questions, and eventually, implementing new
+features, and eventually, closing the ticket.
```
Comment (peterHoburg) [inline docs/TICKETS.md]:
```suggestion
Members of the Core team, and the Team Lead should "work" each ticket submitted, by applying the
proper labels, gathering information, answering questions, implementing new
features, and eventually, closing the ticket.
```

File: docs/TICKETS.md [hunk 2]
```diff
@@ -1,8 +1,85 @@
...
+
+## Labels
+It is important that each new ticket has the correct labels applied to it. This makes it easier
+for contributors to quickly search through for related tickets, or for things to work on.
```
Comment (peterHoburg) [inline docs/TICKETS.md]:
```suggestion
It is important that each new ticket has the correct labels applied to it. This makes it easier
for contributors to quickly search through for related tickets, or find issues to work on.
```

File: docs/TICKETS.md [hunk 3]
```diff
@@ -1,8 +1,85 @@
...
+    - Bug reports
+- `question`
+    - General questions about the project.
+- `duplicate`
+    - Tickets that are closely related to or duplicates of another ticket.
+- `invalid`
+    - A ticket that contains generally invalid or false information, like a bug report that can't
+      be validated, a feature request for a feature that already exists.
+- `triageed`
+    - Tickets that have been confirmed to be implemented into the project.
```
Comment (peterHoburg) [inline docs/TICKETS.md]:
```suggestion
    - Tickets that have been confirmed and will be implemented into the project.
```

File: docs/TICKETS.md [hunk 4]
```diff
@@ -1,8 +1,85 @@
...
+    - Only the Team Lead should apply the `good first issue` label to a ticket.
+- `wontfix`
+    - Tickets that contain enhancements or bug reports that will not be accepted.
+      and added to the project at all.
+    - Only the Team Lead should apply the `wontfix` label to a ticket.
+
+## Assigning and Picking Up Tickets
+### Core Team Members:
+Once a ticket is marked as `triaged`, if you'd like to pick up the ticket to work on implementing it,
+you can assign themselves to the ticket. Do this by clicking the 'assignees' button at the top right,
```
Comment (peterHoburg) [inline docs/TICKETS.md]:
```suggestion
you can assign yourself to the ticket. Do this by clicking the 'assignees' button at the top right,
```

File: docs/TICKETS.md [hunk 5]
```diff
@@ -1,8 +1,85 @@
...
+work on implementing it, post a comment in the ticket saying that you'll be working on it. When a
+Core Team member sees it, they will mark it as assigned to you.
 
-## Submitting Bugs
+Make sure that nobody has asked to be assigned or already been assigned before you begin working,
+to prevent multiple people opening a pull request over the same thing.
 
-TODO
+## Closing Tickets
+Tickets should be closed under the issues raised are resolved. This means:
```
Comment (peterHoburg) [inline docs/TICKETS.md]:
```suggestion
The criteria for ticket closure is as follows:
```

**Review by peterHoburg — APPROVED** [review]

**Comment by Morpheus636** [comment]
Merging as soon as checks pass. Thanks for catching those, they were just errors from the 10+ times I changed the wording.

**Pull request merged**